frontend/src/metabase/entities/groups.js

```diff
@@ -19,21 +19,61 @@ const Groups = createEntity({
     fields: [{ name: "name" }],
   },
 
-  actions: {
-    clearMember: async ({ id }) => {
-      await PermissionsApi.clearGroupMembership({ id });
+  api: {
+    list: (entityQuery, dispatch) =>
+      entityCompatibleQuery(
+        entityQuery,
+        dispatch,
+        permissionApi.endpoints.listPermissionsGroups,
+      ),
+    get: (entityQuery, options, dispatch) =>
+      entityCompatibleQuery(
+        entityQuery.id,
+        dispatch,
+        permissionApi.endpoints.getPermissionsGroup,
+      ),
+    create: (entityQuery, dispatch) =>
+      entityCompatibleQuery(
+        entityQuery,
+        dispatch,
+        permissionApi.endpoints.createPermissionsGroup,
+      ),
+    update: (entityQuery, dispatch) =>
+      entityCompatibleQuery(
+        entityQuery,
+        dispatch,
+        permissionApi.endpoints.updatePermissionsGroup,
+      ),
+    delete: ({ id }, dispatch) =>
+      entityCompatibleQuery(
+        id,
+        dispatch,
+        permissionApi.endpoints.deletePermissionsGroup,
+      ),
+  },
 
-      return { type: CLEAR_MEMBERSHIPS, payload: { groupId: id } };
-    },
+  actions: {
+    clearMember:
+      async ({ id }) =>
```

Comment:
Does this need to be `async`?

```suggestion
     ({ id }) =>
```